src/control/server/mgmt_svc_test.go

```diff
@@ -2138,6 +2139,8 @@ func TestMgmtSvc_StartRanks(t *testing.T) {
 				*srv._superblock.Rank = ioserver.Rank(i + 1)
 			}
 
+			svc.harness.rankReqTimeout = 50 * time.Millisecond
```

Comment:
Maybe would be worth entering a constant default in this test?

Reply:
addressed in https://github.com/daos-stack/daos/pull/1878